tests/scripts/all.sh

```diff
@@ -185,7 +185,7 @@ pre_initialize_variables () {
     export CTEST_OUTPUT_ON_FAILURE=1
 
     # CFLAGS and LDFLAGS for Asan builds that don't use CMake
```

Comment:
It might be worth adding a note that if you want specific optimization flags, you must pass `-O` after `$ASAN_CFLAGS`. I've checked that this is currently always the case.